doc/active_docs/Account Management API.json

```diff
@@ -1,5 +1,55 @@
 {
   "apis": [
+    {
+      "path": "/admin/api/users/{user_id}/access_tokens.json",
+      "responseClass": "List[access_token]",
+      "operations": [
+        {
+          "httpMethod": "POST",
+          "summary": "Access Token Create",
+          "description": "Creates an access token.",
+          "group": "access_token",
+          "parameters": [
+            {
+              "name": "user_id",
+              "description": "ID of the user.",
+              "dataType": "integer",
+              "paramType": "path",
+              "required": true
+            },
+            {
+              "name": "name",
+              "description": "Name of the access token.",
+              "dataType": "string",
+              "required": true,
+              "paramType": "query"
+            },
+            {
+              "name": "permission",
+              "description": "Permission of the access token. It must be either 'ro' (read only) or 'rw' (read and write).",
+              "dataType": "string",
+              "required": true,
+              "paramType": "query"
+            },
+            {
+              "name": "scopes",
+              "description": "Scope of the access token. It is an Array containing one or more of the possible values values. The possible values are, for a master user [\"account_management\", \"stats\"], and for a tenant user [\"finance\", \"account_management\", \"stats\"]",
+              "dataType": "string",
+              "required": true,
+              "paramType": "query"
+            },
+            {
+              "name": "access_token",
+              "description": "A personal Access Token",
```

Comment:
```suggestion
               "description": "Your personal Access Token",
```

Reply:
This is an auto-generated file. The line that causes this is:
https://github.com/3scale/porta/blob/6d7ee3c6ce333a197b41ecec89af64c2da2867a2/app/controllers/admin/api/base_controller.rb#L92
And doing that change would affect **all** docs because this access token is not the one being created here but the one to authenticate to the API.